composer.json

```diff
@@ -13,7 +13,7 @@
     }],
     "require": {
         "league/climate": "^3.0",
-        "symfony/console": "^4.4 || ^5.0",
+        "symfony/console": "^4.4 || ^5.0 || ^6.0",
```

Comment:
You'll need to add support for `maglnet/composer-require-checker` version 4 as well